Simplenote/Verification/AccountVerificationViewController.swift

```diff
@@ -0,0 +1,248 @@
+
+            self.primaryButton.inProgress = false
+            self.secondaryButton.inProgress = false
+            self.updateButtons(isEnabled: true)
+        }
+    }
+
+    func dismiss() {
+        dismiss(animated: true, completion: nil)
+    }
+}
+
+// MARK: - Style
+//
+private extension AccountVerificationViewController {
+    func refreshStyle() {
+        view.backgroundColor = .simplenoteVerificationScreenBackgroundColor
+        iconView.tintColor = .simplenoteTitleColor
+
+        titleLabel.textColor = .simplenoteTextColor
+        titleLabel.font = UIFont.preferredFont(for: .largeTitle, weight: .bold)
+        textLabel.textColor = .simplenoteTextColor
+
+        primaryButton.backgroundColor = .simplenoteBlue50Color
+        primaryButton.setTitleColor(.white, for: .normal)
+        primaryButton.activityIndicatorColor = .white
+        primaryButton.layer.cornerRadius = Constants.primaryButtonCornerRadius
+        primaryButton.titleLabel?.adjustsFontForContentSizeCategory = true
+
+        secondaryButton.backgroundColor = .clear
+        secondaryButton.setTitleColor(.simplenoteTintColor, for: .normal)
+        secondaryButton.titleLabel?.adjustsFontForContentSizeCategory = true
+
+        scrollView.contentInset = Constants.scrollContentInset
+    }
+}
+
+// MARK: - Content
+//
+private extension AccountVerificationViewController {
+    func refreshContent() {
+        let message = String(format: configuration.messageTemplate, email)
+
+        iconView.image = UIImage.image(name: configuration.iconName)
+        titleLabel.text = configuration.title
+        textLabel.attributedText = attributedText(message, highlighting: email)
+        primaryButton.setTitle(configuration.primaryButton, for: .normal)
+        secondaryButton.setTitle(configuration.secondaryButton, for: .normal)
+
+        primaryButton.isHidden = configuration.primaryButton == nil
+    }
+
+    func attributedText(_ text: String, highlighting term: String) -> NSAttributedString {
+        let attributedMessage = NSMutableAttributedString(string: text, attributes: [
+            .foregroundColor: UIColor.simplenoteTextColor,
+            .font: UIFont.preferredFont(forTextStyle: .body)
+        ])
+
+        if let range = text.range(of: term) {
+            attributedMessage.addAttribute(.font,
+                                           value: UIFont.preferredFont(forTextStyle: .headline),
+                                           range: NSRange(range, in: text))
+        }
+
+        return attributedMessage
+    }
+}
+
+// MARK: - Constants
+//
+private struct Constants {
+    static let scrollContentInset = UIEdgeInsets(top: 72, left: 0, bottom: 20, right: 0)
+    static let primaryButtonCornerRadius: CGFloat = 8
+}
+
+// MARK: - Localization
+//
+private struct Localization {
+    struct Review {
+        static let title = NSLocalizedString("Review Your Account", comment: "Title -> Review you account screen")
```

Comment:
this is good! let's all coordinate and see if we can use the same strings for verification